### DISCUSSION TITLE
Reproducing variant prediction results

### DISCUSSION THREAD

**Discussion by devaurs** [facebookresearch/esm#93] [Q&A] [answered]
Hello,
I am interested in using your ESM encoder for protein variant effect prediction. I have read your PNAS article, and I have been very impressed by the results you reported.
I went through your notebook to try and reproduce the results you obtained for beta-lactamase. I had no problem running your code, and I eventually got the same Spearman correlation with true effect scores that you report here (i.e., around 0.8 with SVM). However, at the end of the tutorial you mention that the better results presented in the paper (around 0.89 correlation) were obtained by fine-tuning the model.
I would like to know more about this. Which is the model that you fine-tuned to get these better results: the ESM itself or the regressor that makes the predictions? Was this fine-tuning specific to beta-lactamase, or did it also help with the other proteins you evaluated in the paper? What kind of fine-tuning did you perform?
I would really appreciate if you could share more information about this, as this would be very useful to my own work.
Thank you.

**Comment by tomsercu** [comment, marked as answer]
Finetuning in the deep learning / self-supervised language modeling community usually refers to training through the pretrained model, so updating its weights.
See https://www.pnas.org/content/118/15/e2016239118#sec-18 for the outline of the two settings: either within a protein family, or generalization to new protein families.

**Reply by devaurs** [reply]
Thank you for your prompt reply!
